includes/REST_API/Font_Controller.php

```diff
@@ -0,0 +1,470 @@
+<?php
+/**
+ * Class Font_Controller
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\REST_API;
+
+use WP_Error;
+use WP_Post;
+use WP_Query;
+use WP_REST_Posts_Controller;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_REST_Server;
+
+/**
+ * Font_Controller class.
+ */
+class Font_Controller extends WP_REST_Posts_Controller {
+	/**
+	 * Registers the routes for posts.
+	 *
+	 * @since 1.16.0
+	 *
+	 * @see register_rest_route()
+	 *
+	 * @return void
+	 */
+	public function register_routes() {
+		register_rest_route(
```

Comment:
Will we support batching here?

Reply:
There are no UI features that would require any batching capabilities